magento/pwa-studio
# Initial draft of prex integration with PWA

## Description

This PR adds documentation for how to integrate product recommendations with PWA.

## Related Issue

Closes JIRA-DATA-2240

## Acceptance



### Verification Stakeholders
@jcalcaben 

### Verification Steps
1. Go to the https://magento.github.io/pwa-studio/ site.
2. Verify that under the **Integrations** section, **Product Recommendations** appears.
3. Click through the **Product Recommendations** page to ensure the content matches what was requested in the JIRA ticket.

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
